packages/amplify-graphql-auth-transformer/src/__tests__/rds-query-auth.test.ts

```diff
@@ -1,12 +1,29 @@
 import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
-import { validateModelSchema } from '@aws-amplify/graphql-transformer-core';
+import { constructDataSourceStrategies, validateModelSchema } from '@aws-amplify/graphql-transformer-core';
 import { testTransform } from '@aws-amplify/graphql-transformer-test-utils';
 import { parse } from 'graphql';
-import { AppSyncAuthConfiguration, SQLLambdaModelProvisionStrategy } from '@aws-amplify/graphql-transformer-interfaces';
-import { AuthTransformer } from '../graphql-auth-transformer';
+import {
+  AppSyncAuthConfiguration,
+  SQLLambdaModelDataSourceStrategy,
+  SqlModelDataSourceDbConnectionConfig,
+} from '@aws-amplify/graphql-transformer-interfaces';
 import { PrimaryKeyTransformer } from '@aws-amplify/graphql-index-transformer';
+import { AuthTransformer } from '../graphql-auth-transformer';
 
 describe('Verify RDS Model level Auth rules on queries:', () => {
+  const dbConnectionConfig: SqlModelDataSourceDbConnectionConfig = {
+    hostnameSsmPath: '/test/hostname',
+    portSsmPath: '/test/port',
+    usernameSsmPath: '/test/username',
+    passwordSsmPath: '/test/password',
+    databaseNameSsmPath: '/test/databaseName',
+  };
+  const mysqlStrategy: SQLLambdaModelDataSourceStrategy = {
+    name: 'mysqlStrategy',
+    dbType: 'MYSQL',
+    dbConnectionConfig,
+  };
```

Comment:
This seems repetitive and can be abstracted out.

Reply:
as above, I'll take it on in the upcoming combine PR